### DISCUSSION TITLE
Model Tuning

### DISCUSSION THREAD

**Discussion by fxpeterbs31** [teamalgoritma/community#280] [Q&A] [answered]
Halo, saya mau tanya. Dalam pemodelan pada machine learning, kita mengenal istilah model tuning. Apakah sebaiknya kita dalam melakukan pemodelkan hanya menggunakan 1 model (model hasil tuning saja, model awal sebelum tuning tidak perlu dipakai) atau keduanya digunakan?

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @fxpeterbs31 ,

Pertanyaan yang menarik. Apakah kita hanya menggunakan model tuning saja? Sebelum itu, kita telah paham bahwa tuning model dilakukan untuk meningkatkan performa model yang kita bangun. Terkadang setelah melakukan tuning hasil yang diinginkan tidak terpenuhi dan bisa saja tidak meningkat signifikan performanya dibanding model sebelum tuning.
Sehingga, dilakukan tuning model agar kita mempunyai lebih banyak pilihan model yang dapat digunakan, baik hasil tuningnya bisa lebih baik ataupun tidak. 
Kembali ke pertanyaan awal, apakah hanya digunakan model hasil tuning saja? Yaa, biasanya jika model hasil tuningnya jauh lebih baik, kita hanya menggunakan model setelah tuning demi mendapatkan performa dan prediksi terbaik.

Sekian jawaban dari kami, semoga membantu.
